### DISCUSSION TITLE
Sorting of filter criterias as sorted in term store

### DISCUSSION THREAD

**Discussion by HeKr09** [microsoft-search/pnp-modern-search#1880] [Ideas]
**Is your feature request related to a problem? Please describe.**
Currently you are able to sort the filter values within a refinement by name or by count. Couldn't there be a possibility to sort it as in the term store?

**Describe the solution you'd like**
Sorting of values as in the term store order.

**Describe alternatives you've considered**
none found :(

**Comment by wobba** [comment]
@HeKr09 if you can provide the code which can read the sort order from the term store, which I expect might be related to the term store locale, we could possibly improve this. That said, in a search result we have no way of knowing what source the data come from and then tie back to use the correct sorting.

Sorting values based on the UX locale is probably the closest we can get.

**Reply by FranckyC** [reply]
@HeKr09 the way it works here does not allow to do this as filter values are not necessarily taxonomy terms. To do what you want, you need a Taxonomy picker filter. See https://github.com/microsoft-search/pnp-modern-search/discussions/1366
